File: Iniciante/1012.py
A, B, C = input().split()

tri = (float(A) * float(C)) / 2
cir = 3.14159 * float(C)**2
tra = ((float(A) + float(B)) * float(C)) / 2
qua = float(B)**2
ret = float(A) * float(B)

print(f'TRIANGULO: {tri:.3f}\nCIRCULO: {cir:.3f}\nTRAPEZIO: {tra:.3f}\nQUADRADO: {qua:.3f}\nRETANGULO: {ret:.3f}')

File: Iniciante/1012.php
<?php

$l1 = explode(" ", readline());
$a = $l1[0];
$b = $l1[1];
$c = $l1[2];

$tri = number_format($a*$c/2, 3, '.', '');
$cir = number_format(3.14159*$c**2, 3, '.', '');
$tra = number_format(($a+$b)*$c/2, 3, '.', '');
$qua = number_format($b**2, 3, '.', '');
$ret = number_format($a*$b, 3, '.', '');

echo "TRIANGULO: $tri\nCIRCULO: $cir\nTRAPEZIO: $tra\nQUADRADO: $qua\nRETANGULO: $ret\n";

?>